Start-C4BNexusSteup.ps1

```diff
@@ -939,7 +939,7 @@ function New-NexusRawComponent {
 }
 
 # Install base nexus-repository package
-choco install nexus-repository -y
+choco install nexus-repository -y --source="'https://chocolatey.org/api/v2/'"
```

Comment:
Should point to `https://community.chocolatey.org/api/v2. This is fine now while we have the alias in place, but eventually this will need changed. Might as well do it now.

Reply:
Good point! I'll address this in the next commit.